k8s/k8s.go

```diff
@@ -117,7 +182,7 @@ func CmdAddK8s(args *skel.CmdArgs, conf utils.NetConf, hostname string, calicoCl
 		endpoint.Metadata.Node = hostname
 		endpoint.Metadata.Orchestrator = orchestrator
 		endpoint.Metadata.Workload = workload
-		endpoint.Metadata.Labels = make(map[string]string)
+		endpoint.Metadata.Labels = labels // Only when policy type == k8s
```

Comment:
Not a big fan of inline comments like this.  It's also not a very useful one, so I'd recommend just removing it.